tests/integration/static_ip_address_test.py

```diff
@@ -257,3 +277,119 @@ def test_add_static_ipv6_with_no_address():
     eth1_cur_state = cur_state[INTERFACES][0]
     # Should have at least 1 link-local address.
     assert len(eth1_cur_state['ipv6']['address']) >= 1
+
+
+def test_add_static_ipv6_with_min_state():
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'eth2',
+                'type': 'ethernet',
+                'state': 'up',
+                'ipv6': {
+                    'enabled': True,
+                    'address': [
+                        {'ip': IPV6_ADDRESS1, 'prefix-length': 24}
+                    ]
+                }
+            }
+        ]
+    }
+    netapplier.apply(copy.deepcopy(desired_state))
+
+    assertlib.assert_state(desired_state)
+
+
+@pytest.mark.xfail
```

Comment:
Better give a reason and make it strict so we will not miss it when we fix the problem.

Reply:
Done